.yarn/patches/@metamask-providers-npm-17.2.0-cb0cfcdadc.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/dist/extension-provider/createExternalExtensionProvider.mjs b/dist/extension-provider/createExternalExtensionProvider.mjs
+index f2aa9285d84cd5e25c20f5202437c476cdacc326..121a9099e456d3444d8aae79574ba9c53e7f2699 100644
+--- a/dist/extension-provider/createExternalExtensionProvider.mjs
++++ b/dist/extension-provider/createExternalExtensionProvider.mjs
+@@ -1,5 +1,4 @@
+-import $detectbrowser from "detect-browser";
+-const { detect } = $detectbrowser;
++import { detect } from "detect-browser";
+ import $extensionportstream from "extension-port-stream";
+ const { PortDuplexStream: PortStream } = $extensionportstream;
+ import config from "./external-extension-config.json" assert { type: "json" };
```

Comment:
without this patch, I get the following from the inpage provider in the dapp web console

```
createExternalExtensionProvider.mjs:2 Uncaught TypeError: Cannot destructure property 'detect' of 'detect_browser__WEBPACK_IMPORTED_MODULE_0__.default' as it is undefined.
    at 193000 (createExternalExtensionProvider.mjs:2:9)
    at __webpack_require__ (bootstrap:19:1)
    at 701043 (createExternalExtensionProvider.mjs:49:61)
    at __webpack_require__ (bootstrap:19:1)
    at make namespace object:7:1
    at inpage.js:70:1
    at inpage.js:70:1
 ```